cms/static/js/spec/utils/drag_and_drop_spec.js

```diff
@@ -146,8 +146,8 @@ define(["js/utils/drag_and_drop", "common/js/components/views/feedback_notificat
                         left: $ele.offset().left
                     });
                     destination = ContentDragger.findDestination($ele, 1);
-                    expect(destination.ele).toBe($('#subsection-list-2'));
-                    expect(destination.parentList).toBe($('#subsection-2'));
+                    expect(destination.ele[0]).toBe($('#subsection-list-2')[0]);
+                    expect(destination.parentList[0]).toBe($('#subsection-2')[0]);
```

Comment:
yeah we could and take advantage of jasmine-jquery matchers but two jquery libraries messes this up. Any other reason to use `toEqual`?